Makefile

```diff
@@ -23,9 +23,37 @@ install-go-tools: mod-download
 	go install github.com/onsi/ginkgo/ginkgo@v1.16.4
 	go install golang.org/x/tools/cmd/goimports
 
+# proto compiler installation
+PROTOC_VERSION:=3.15.8
+PROTOC_URL:=https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}
+.PHONY: install-protoc
+install-protoc:
+	if [ $(shell ${DEPSGOBIN}/protoc --version | grep -c ${PROTOC_VERSION}) -ne 0 ]; then \
+		echo expected protoc version ${PROTOC_VERSION} already installed ;\
+	else \
+		if [ "$(shell uname)" == "Darwin" ]; then \
+			echo "downloading protoc for osx" ;\
+			wget $(PROTOC_URL)-osx-x86_64.zip -O $(DEPSGOBIN)/protoc-${PROTOC_VERSION}.zip ;\
```

Comment:
is there a mac arm64 version too?

Reply:
great question, so there isn't a specific arm version to download for mac surprisingly. This one works on mac arm too so it works out of the box for all our macs.